packages/lodestar-cli/src/util/file.ts

```diff
@@ -31,7 +32,7 @@ export function stringify<T=Json>(obj: T, fileFormat: FileFormat): string {
       contents = JSON.stringify(obj, null, 2);
       break;
     default:
-      throw new Error("Invalid filetype");
+      throw new YargsError("Invalid filetype");
```

Comment:
This is not an "expected" error, I would throw a regular error

Reply:
done